telmos_addins.py

```diff
@@ -118,30 +110,12 @@ def telmos_addins(delta_root, tmfs_root,
         if te_array.nbytes < 500:
             log_func("Addin TE array is incomplete: %s" % f_key)
             
-        if do_output:
-            out_file = os.path.join(tmfs_root, "Runs", tel_year, "Demand",
-                                    tel_id, filename.replace(".DAT", "TE.DAT"))
-            format_string = ["%d"] + ["%.3f" for x in range(te_array.shape[1]-1)]
-            np.savetxt(out_file, te_array, delimiter=",", fmt=format_string)
-            log_func("Saved Trip Ends to %s" % str(out_file))
-            produced_files.append(out_file)
-            file_names.append(filename.replace(".DAT", "TE.DAT"))
-            
-    # Check against supplied files
-    if debug is True:
-        log_func("\nChecking against previous files in directory 'Received Data'\n")
-        diffs = {k:0 for k in file_names}
-        check_path = os.path.join("Received Data", "Output", "37DSL_out")
-        check_files = [os.path.join(check_path, f) for f in file_names]
-        for name, p, c in zip(file_names, produced_files, check_files):
-            skiprows = 0
-            p_total = np.loadtxt(p, skiprows=skiprows, delimiter=",").sum()
-            c_total = np.loadtxt(c, skiprows=skiprows, delimiter=",").sum()
-            diffs[name] = (p_total - c_total) / c_total
-        return diffs
-                
-                
-                
-                
+        out_file = os.path.join(tmfs_root, "Runs", tel_year, "Demand",
+                                tel_id, filename.replace(".DAT", "TE.DAT"))
+        format_string = ["%d"] + ["%.3f" for x in range(te_array.shape[1]-1)]

```

Comment:
Convention is to use `_` for throwaways

```suggestion
        format_string = ["%d"] + ["%.3f" for _ in range(te_array.shape[1]-1)]
```